MANIFEST.in

```diff
@@ -1,5 +1,5 @@
-include pytorch_lightning/py.typed  # marker file for PEP 561
-include CHANGELOG.md
-recursive-include requirements *.txt
-include requirements.txt
+exclude *.toml  # project config
+exclude requirements.txt
+exclude __pycache__
+include .actions/setup_tools.py
```

Comment:
Why do we need to include this script in the release when it's only used by us at build time?

Reply:
it turned out that when you install a package from a build package it is also called setup and then it is needed, but we can bypass it by adding an extra file in the package indication we run the prepared package...